api/src/services/member/member.service.ts

```diff
@@ -15,7 +15,6 @@ import {
 import { Member, Place } from '../../types/models';
 import { MemberInfoView } from '../../types/views';
 import { SessionInfo } from 'session-info.interface';
-import { Request, Response } from 'express';
```

Comment:
This lines needs to be re-added in

```
import { Request, Response } from 'express';
```

Reply:
it also needs reverting on member.routes.ts. I accidentally suggested it to be removed in my initial review.